client/src/routes/routes-list.tsx

```diff
@@ -79,4 +79,113 @@ const routesList: RouteObject[] = [
 	},
 ]
 
-export default routesList
+export const routesListMentor: RouteObject[] = [
+	{
+		element: <Layout />,
+		children: [
+			{
+				path: _URL.HOME,
+				element: (
+					<ProtectedRoute>
+						<Navigate to="/meetings" />
+					</ProtectedRoute>
+				),
+			},
+			{
+				path: _URL.LOGIN,
+				element: <Login />,
+			},
+			{
+				path: _URL.LOGOUT,
+				element: <Logout />,
+			},
+			{
+				path: _URL.MEETINGS,
+				element: (
+					<ProtectedRoute>
+						<MeetingsProvider>
+							<Meetings />
```

Comment:
this page should work a bit differently for mentors. 
it should search through all meetings we have, but check for the name in the `mentor` cell, not in `stunents`. 

because now it shows the list of the meetings this mentor "attended"
![image](https://github.com/BUI-studies/buoy-platform/assets/104223028/9e2ec993-c0ec-4942-beab-3bd04fb7b6d9)

Reply:
fixed in terms of code, but needs a testing after the tables would be changed